src/claimtrie.h

```diff
@@ -23,6 +28,9 @@
 #define SUPPORT_QUEUE_NAME_ROW 'p'
 #define SUPPORT_EXP_QUEUE_ROW 'x'
 
+// hard fork height for claim name normalization
+static const int LBRY_NORMALIZED_NAME_FORK_HEIGHT = 329320;
+
```

Comment:
We should make this adjustable via chainparams.cpp so that we can define different block height for testing in regtest/testnet